06_iito/index.tex

```diff
@@ -42,17 +42,17 @@ \section{Implementation}
 \subsection{Server Client Infrastructure}
 \label{subsec:IITO:server_client_infrastructure}  
 
-All IITO sessions, such as Gift Giving or Common Pool Donations, follow the same client server interaction format. The server acts a middleman for all agent to agent communication, and prompts agents to make decisions or formulate messages during the session. At each stage in a session the server compiles a list of messages from the agents, following that sessions' protocols, and then uses that list as input for the next stage. While a bit abstract this is better explained through an example.
+During a turn, once IITO has started, a series of sessions are run and the conclusion of all these sessions indicates that IITO is also complete. In the current implementation these sessions include gift giving and common pool donations and they all follow the same client server interaction format\footnote{Server and Client are defined in Definition~\ref{def:server} and Definition~\ref{def:client} respectively}. The server acts an intermediary for all client to client communication, and prompts clients to make decisions or formulate messages during the session. At each stage in a session the server compiles a list of messages from the clients, following that sessions' protocols, and then uses that list as input for the next stage. While a bit abstract this is better explained through an example.
```

Comment:
suggested re-writing:
During a turn, once IITO has started, a series of sessions are run**,** and the conclusion of all these sessions indicates that IITO is also complete. In the current implementation**,** these sessions include gift giving and common pool donations, and they all follow the same **client-server** interaction.